From: Antoine Musso <hashar@free.fr>
Date: Wed, 22 Jul 2015 23:01:16 +0200
Subject: Revert "Fix passing labels to Gerrit when they are not defined in
 All-Projects"

That changed Zuul to send `gerrit review` with `--label verified=1`
which causes a SQL backend error on our side (T106596).  It should be
upper case first `Verified` but setting that in the layout.yaml file
causes approval to no more matches cause they are made against a lower()
case version.

Revert back to passing --code-review and --verified, that will let us
switch back our layout.yaml file back to all lower case which fits the
approval matching rules.

This reverts commit c387a5043e9db2ec87699ea5fd2b08b40d12e882.

Bug: T106531
Bug: T106596
Change-Id: I74fa0367605bf395df0380c81468c96f0b5b6e84
---
 zuul/lib/gerrit.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zuul/lib/gerrit.py b/zuul/lib/gerrit.py
index 1434389..d2e155c 100644
--- a/zuul/lib/gerrit.py
+++ b/zuul/lib/gerrit.py
@@ -120,7 +120,7 @@ class Gerrit(object):
             if v is True:
                 cmd += ' --%s' % k
             else:
-                cmd += ' --label %s=%s' % (k, v)
+                cmd += ' --%s %s' % (k, v)
         cmd += ' %s' % change
         out, err = self._ssh(cmd)
         return err
